source/likely.js

```diff
@@ -2,5 +2,6 @@
 
 var likely = require('./index.js');
 
-window.likely = likely;
 window.addEventListener('load', likely.initiate);
```

Comment:
I’m here, sorry for the delay.

I’m very doubtful about this line. If we switch to UMD build for all environments, this will make likely auto-initialize when used [as a CommonJS module](https://github.com/ilyabirman/Likely#using-as-a-commonjs-module) which breaks the current behavior.

We can break the current behavior and release a new major, but I don’t want to do it:
* We can make Likely auto-initialize even when used as a CommonJS module, but it’s a weird thing because a simple `require('ilyabirman-likely')` will cause side effects.
* We can make Likely always require manual initialization, but it’ll make browser users’ life harder which is also bad.

Instead, we should find a way to make Likely auto-initialize itself when being loaded as a separate script. The only reliable solution I can think of is to build two bundles, one for browser and one for CommonJS env, and point `main` in `package.json` to the CommonJS bundle. @vitkarpov, @andrew--r, probably you have any other ideas?